TestProject.BL/Services/PostService.cs

```diff
@@ -150,5 +158,10 @@ private void SetSelectedRating(int currentUserId, List<PostModel> postModelList)
                     ? RatingValue.Unrated : selectedRating.Value;
             }
         }
+
+        private bool ValidatePost(EditPostModel post)
+        {
+            return (!string.IsNullOrEmpty(post.Title)) && (!string.IsNullOrEmpty(post.Content));
```

Comment:
Why don't you use asp.net validation?
https://docs.microsoft.com/en-us/aspnet/core/mvc/models/validation?view=aspnetcore-3.1

Reply:
I forgot about it